_posts/2017-06-13-interview-with-eddy-colloton.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title:  "Interview with Eddy Colloton"
+date:   2017-06-13
+categories: mediaconch, interview
+---
+
+# Interview with Eddy Colloton
+
+*Editor's note: This is the first in a series of interviews with people using MediaConch within their institutions. Enjoy!*
+
+
+**Hey Eddy! Introduce yourself please.**
+
+Hey Ashley! I’ve recently become a “Denverite” and have started a new job as an [Assistant Conservator specializing in electronic media](https://www.google.com/url?q=http://www.kunc.org/post/built-past-floppy-disk-and-vhs-art-need-creative-conservation&sa=D&ust=1497302411695000&usg=AFQjCNEd4gxsXMGMqdLlhnkyQ3U_cuCpug) at the Denver Art Museum (DAM). Before that, I was down in Baton Rouge, Louisiana working as a [National Digital Stewardship Resident](https://www.google.com/url?q=https://ndsr.americanarchive.org/&sa=D&ust=1497302411696000&usg=AFQjCNFAxhriwjXWqTbSm04mdYPhmQx3aA) with Louisiana Public Broadcasting (LPB). When I’m not working, I like to listen to podcasts, read comics, play bass, and stare into the endless void that is “twitter” ([@EddyColloton](https://twitter.com/EddyColloton)). I’m on a big H. P. Lovecraft kick right now so let me know if you have any recommendations (Dunwich Horror is my current fav, but At the Mountains of Madness is a close second).
```

Comment:
"https://www.google.com/url?q=" and all including and after "&" in the URL are not needed (slower due to redirection, more private info provided to Google), should be removed everywhere.